### ISSUE TITLE
Accept the default fixation in the soil profile

### ISSUE THREAD

**Issue by byzheng** [APSIMInitiative/ApsimX#9122]
### Describe the new feature

I am run simulations using soil dataset download from gridded dataset, ASRIS. In some few simulations, I get errors about soil water profile, e.g. 

```
Saturation of 0.469 in layer 2 is above acceptable value of 0.468. You must adjust bulk density to below 1.406 OR saturation to below 0.468
Saturation of 0.475 in layer 3 is above acceptable value of 0.474. You must adjust bulk density to below 1.391 OR saturation to below 0.474
```

I understand I can manually modify the soil profile to fix this problem. However, it will be time-consuming to check and modify large amount apsimx file in the grid simulations. 

Could we accept the default fixation (e.g. saturation) to ignore this error?

- [ ]

**Comment by hut104** [comment]
I wonder if a good fix to this would include getting ASRIS to avoid soil specifications that are not physically possible.
The examples included here look to be very close (ie small errors in precision I'd imagine).

**Comment by byzheng** [comment]
@hut104 I agree with you. It might be caused by the float error at rounding to 3 digits in ASRIS or APSIMX
